src/mainframe/models/schemas.py

```diff
@@ -16,6 +19,57 @@ class Error(BaseModel):
     detail: str
 
 
+class Package(BaseModel):
+    """Model representing a package queried from the database."""
+
+    scan_id: str
+    name: str
+    version: str = ""
+    status: str = ""
+    score: Optional[int] = 0  # Just for the sake of texting, remove upon merge
+    inspector_url: Optional[str] = ""
+    rules: list[str] = []
+    download_urls: list[str] = []
+    queued_at: str = ""
+    queued_by: str = ""
+    pending_at: Optional[str] = None
+    pending_by: Optional[str] = None
+    finished_at: Optional[str] = None
+    finished_by: Optional[str] = None
+    commit_hash: Optional[str] = ""
```

Comment:
Appears to be missing `reported_at` and `reported_by` field. I'd like for this information to be available as well so the client can tell if it's been reported already.